charts/quickwit/values.yaml

```diff
@@ -128,7 +237,41 @@ config:
   # searcher:
   #   fast_field_cache_capacity: 10G
   #   split_footer_cache_capacity: 1G
-  #   max_num_concurrent_split_streams: 100  
+  #   max_num_concurrent_split_streams: 100
+
+  index: {}
```

Comment:
This part is very related to Contentsquare, this is too specific to be in this repository.